### PR TITLE
SYM-108

### PR DESCRIPTION
**Pull request by djordje-mladenovic**
Fields for inputHeight and inputWidth as well as elementHeight and elementWidth are exposed for both BT and VGS PIN fields

### CODE REVIEW

**Review by devinmorgan — CHANGES_REQUESTED** [review]
Nice work so far, Djordje. Here are a couple of changes I was hoping you could make:
1. Are the usages of `this` necessary in this PR? When I played around with it locally, they could be dropped. For example, prefer `getDemiensionPixelSize(...)` instead of `this.getDemiensionPixelSize(...)` to conform to the current conventions
2. The lint check seems to be failing. Hopefully, all of the lining issues will be fixed if you run `./gradlew spotlessApply`. This is mentioned in [the README_DEV.md](https://github.com/teamforage/forage-android-sdk/blob/687f8661442f82e96dd26bbfd2f5748e6b000224/README_DEV.md?plain=1#L91) for your reference.
3. Can you please include screenshots showcasing this [ticket's acceptance criteria](https://linear.app/joinforage/issue/SYM-108/android-styling-changes-enable-customization-of-styling-of-pin-input):
    - [x] BasisTheory - PIN Input field's width is smaller than the Forage logo's width 
    - [x] VGS - PIN Input field's width is smaller than the Forage logo's width 
    - [ ] BasisTheory - PIN Input field's width and height are much larger than normal
    - [ ] VGS - PIN Input field's width and height are much larger than normal
4. Can you please update the catalog view of the Sample App by altering the appearance of one of the existing `ForagePINEditText` elements and one of the `ForagePANEditText` elements so that they have non-default values for:
    - [ ] `inputHeight`
    - [ ] `inputWidth`
    - [ ] `elementHeight`
    - [ ] `elementWidth`
    Changing the catalog view in the Sample App will show us how to use the changes you've made so that we can update our documentation accordingly. It will also help us test ourselves locally.

**Comment by djordje-mladenovic** [comment]
BTPinFieldSmallerThanLogo
BTPinFieldLargerThanNormal
VGSPinFieldLargerThanNormal
VGSPinFieldSmallerThanLogo

File: sample-app/src/main/res/layout/fragment_catalog.xml
Comment (devinmorgan) [inline sample-app/src/main/res/layout/fragment_catalog.xml:1]: Thank you for these changes! It makes it much more clear how styling works in Android

**Review by devinmorgan — APPROVED** [review]
Thanks for your following changes @djordje-mladenovic! I'll make a follow-up PR that updates the `README.md`. These changes seem to be what we needed!

**Pull request merged**